Requirements.md

```diff
@@ -0,0 +1,19 @@
+## Software Requirements
```

Comment:
Overall, you did a good job writing requirements. There may be some ambiguity about how you will know if the requirements are met or not. For example, what is supposed to happen if a merge is aborted by "accepting an abort option"? I assume successfully auto-merged changes are kept, but non-automatic merges are abandoned/reverted? I've never used this feature of Git. It may be enough to provide a link to Git documentation and reference the documentation once at the top of the document.

Reply:
That's a good point, I will add them.